---
title: Research Data
category: Research-Data-Management
layout: default
docs_css: markdown
redirect_from: /Research-Data-Management
---

## Definition of Research Data
There is no consensus on the definition of research data as they are highly heterogeneous. Thus, the definition can vary considerably depending on the research funder, the scientific discipline or subject, and the research data itself {% cite lindst√§dt_2019 biernacka:2020 voigt_2022 %}. We propose the following definition, based on around 20 others: **research data** is the collection of digital and non-digital objects (excluding scientific publications) that are generated (e.g. through measurements, surveys, source work), studied and stored during or as a result of scientific research activities. These objects are commonly accepted in the scientific community as necessary for the production, validation and documentation of original research results. In the context of Research Data Management (RDM), research data also includes non-data objects such as software and simulations (see further examples below).

The characteristics of research data depend strongly on the context (i.e. conditions of generation, methods used, perspective) {% cite biernacka:2020 %}. Nevertheless, we can try to classify them as follows:

**Primary** or **raw data** is potential information generated by a researcher for the first time during a research project. It is unprocessed, possibly even untouched by human hands, unseen by human eyes, unthought by human minds. It needs to be contextualised to make it accessible to the human audience {% cite pomerantz:2015 darby:2021 golmand_and_martin:2023 %}. This data can be further categorised as **observational** (e.g. archaeological samples, brain scans, opinion polls), **experimental** (e.g. clinical trial data, DNA sequencing or organic material) and **simulation**, which is the modelling of complex processes (e.g. climate simulations) {% cite darby:2021 %}.

**Secondary data** is data compiled from existing sources. This includes **derived** or **compiled data** (e.g. corpora, databases created by extracting information from multiple secondary sources) {% cite darby:2021 %}. 

Then there are **processed data** (i.e. raw data made useful {% cite golmand_and_martin:2023 %}), **analysed data** (i.e. processed data that has been interpreted {% cite golmand_and_martin:2023 %}), **finalised**, **published** or **reference data** (i.e. curated data that support your research question  {% cite golmand_and_martin:2023 %}, such as gene banks, national statistical archives {% cite darby:2021 %}) and **information about the means** necessary to generate data or replicate results (e.g. computer code, experimental methods) {% cite darby:2021 %}.

Data is differentiated from **information** (i.e. processed data that can be consumed by humans), **knowledge** (i.e. information that has been assimilated by humans) and **wisdom** (i.e. applied knowledge) {% cite gerlich:2023 %}. 

![Information pyramid]({{ '/assets/img/information_pyramid.png' | relative_url }} "Information pyramid"){:width="70%"}

## General Data Types
---
 General data types include the following {% cite NC_State_University_Library:2020 steen:2022 voigt_2022 dfg:2015 %}:
* Data files (e.g. text files, binary files)
* Documents (e.g. word processing documents, spreadsheets)
* Measurement data, lab and observation data
* Lab and field notebooks, diaries
* Questionnaires, transcripts, codebooks
* Survey data
* Audio and video tapes
* Spectra
* Test answers
* Slides, artefacts, specimens, samples
* Database content (e.g. text, video, audio, images)
* Models, simulations, algorithms, scripts, code
* Content of an app (e.g. software) and research software
* Methodologies and workflows
* Standard Operating Procedures (SOPs) and protocols

## Common Data Types in Microbiology
---
Data types in microbiology include the following:
* Antibiotic resistance data
* Biochemical assay data
* Clinical data
* Crystallographic data
* Geospatial data
* Image data
* Linked genotype and phenotype data (e.g. personally identifiable genetic and phenotypic data resulting from biomedical research projects)
* Linked Open Data (LOD)
* Macromolecular structures (e.g. electron microscopy data)
* Metabolomes
* Microbiome data (e.g. physical microbiome interactions)
* Nucleic acid sequences (e.g. raw sequencing data (reads or traces), amplicon, genome assemblies, annotated sequences) such as:
  * DNA sequences
    * (Meta)genomes
    * Metagenome Assembled Genomes (MAGs)
    * Genetic polymorphism
    * Genomic features
    * Genomic organisation
    * Epigenomic data
  * RNA sequences
    * 16S, 18S and ITS ribosomal RNA sequences
    * Functional genomics / gene expression data (e.g. ribosome profiling, from microarrays)
    * RNA-protein interactions
    * Small RNA (sRNA)
    * (Meta)transcriptomes
  * Genetic variation data
* Protein sequences
  * Protein-protein interactions
  * (Meta)proteomes (e.g. mass spectrometry-based proteomics data, including peptide and protein expression information (identifications and quantification values) and the supporting mass spectra evidence)
* Quantitative and predictive food microbiology
* Sample and project (meta)data
* Scientific texts
* Semantic data
* Species interaction data (e.g. physical microbial interaction data, host-microbe interaction data)
* Standardised bacterial information
* Vertebrate-virus network
