---
title: Data Repositories
category: RDM-Share
layout: default
docs_css: markdown
---
A **data repository** is a location where digital objects are stored and documented, and which enable the separate publication and archiving of these objects. Data access can be either open or restricted to a group of users.

## Benefits of publishing data in a repository
* **Back-up** (i.e. the practice of keeping additional copies of your research data in a separate physical or cloud locations from your files in storage).
* Increased **data discoverability** and **accessibility**, and thus potential **data reuse**.
* Increased **visibility** and **citations** as repositories can be harvested by **search engines** such as:
    * [BASE](https://www.base-search.net/about/en/index.php)
    * [DataCite Search](https://search.datacite.org/)
    * [Google Dataset Search](https://datasetsearch.research.google.com/)
    * [LIVIVO - ZB MED Search Portal for Life Sciences](https://www.livivo.de/)

## Criteria to select a trusted repository
Below are listed criteria you might want to consider when selecting a repository where to publish your data:
* Is the repository **certified** according to CoreTrustSeal (CTS), nestor seal/DIN 31644 or ISO 16363? Find these repositories via re3data or the website of various certification initatives:
    * [Find CTS-certified repositories on re3data](https://www.re3data.org/search?query=&certificates%5B%5D=CoreTrustSeal)
    * [Find nestor/DIN 31644-certified repositories on re3data](https://www.re3data.org/search?query=&certificates%5B%5D=DIN%2031644)
    * [Find ISO16363-certified repositories on re3data](https://www.re3data.org/search?query=&certificates%5B%5D=ISO%2016363)
    * [Find a CTS-certified repository on the CTS-Website](https://amt.coretrustseal.org/certificates)
    * [Find a nestor/DIN 31644-certified repository on the nestor-Website](https://www.langzeitarchivierung.de/Webs/nestor/EN/Zertifizierung/nestor_Siegel/siegel.html)
    * [Find an ISO16363-certified repository on the CTAB-Website](http://www.iso16363.org/iso-certification/certified-clients/)
* Is the repository among the repositories preferred by a **funder** and/or endorsed by the **international research community**? In general, community-endorsed repositories should be well known, well-used and have a good continuity plan. If they existed for a long time and contain a lot of published data, their long-term sustainability is likely covered.
* Does the repository provide:
    * **Open Access** to non-sensitive data?
    * The **terms of use** and **licenses** of the data?
    * A **policy** to help researchers determine whether their data needs will be met?
* Does the repository use a Persistent Identifier (PID) system such as assigning a **Digital Object Identifier** (DOI) to submitted datasets?
* Does the repository offer the possibility to integrate all metadata relevant for finding and specifying your publication? 
* Further criteria: costs, repository size, data upload restriction(s), landing pages, guidance on data citation.

## Order of preference for selecting a repository
1. A certified resposity (CoreTrustSeal, nestor/DIN 31644, ISO 16363)
2. A well-established repository in your discipline that offers a specific scope (e. g. supporting a specific metadata schema).
3. One of the repositories recommended by your funding organization or the funding program.
4. Your institution’s repository (if available).
5. A cost-free interdisciplinary repository (e.g. [Figshare](https://figshare.com/), [Zenodo](https://zenodo.org/)).
6. Another repository that you can search for using the above-mentioned criteria in a repository finder. 

## Well-established repositories for data deposition in microbiology
Below are listed well-established repositories in microbiology. For each repository, the FAIRsharing and re3data pages are linked. On the FAIRsharing page, you will find information such as which journals endorse the repository (under "Collections & Recommendations" and then "In Policies"). On the re3data page, you will find information such as the above-mentioned criteria to select a trusted repoository.

| Data type 	| Data repository 	| FAIRsharing 	| re3data 	| 
|---	|---	|---	|---	|
| **All research output** 	| [Zenodo](https://zenodo.org/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.wy4egf) 	| [re3data](https://www.re3data.org/repository/r3d100010468) 	|
| **Crystallographic data** 	| Cambridge Structural Database ([CSD](https://www.ccdc.cam.ac.uk/solutions/csd-core/components/csd/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.vs7865) 	| [re3data](https://www.re3data.org/repository/r3d100010197) 	|
| **Geospatial data** 	| Malaria Atlas Project ([MAP](https://malariaatlas.org/)) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100011838) 	|
| **Image data** 	| [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.x38D2k) 	| [re3data](https://www.re3data.org/repository/r3d100013949) 	|
|  	| [Cell Image Library](http://www.cellimagelibrary.org/home) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.8t18te) 	| [re3data](https://www.re3data.org/repository/r3d100000023) 	|
| **Linked genotype and phenotype data** 	| European Genome-phenome Archive ([EGA](https://ega-archive.org/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.mya1ff) 	| [re3data](https://www.re3data.org/repository/r3d100011242) 	|
| **Macromolecular structures** 	| Worldwide Protein Data Bank ([wwPDB](http://www.wwpdb.org/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.mckkb4) 	| [re3data](https://www.re3data.org/repository/r3d100011104) 	|
|     | RCSB Protein Data Bank ([RCSB PDB](https://www.rcsb.org))     | [FAIRsharing](https://fairsharing.org/FAIRsharing.2t35ja) 	| [re3data](https://www.re3data.org/repository/r3d100010327) 	|
|     | Protein Data Bank of Japan ([PDBj](https://pdbj.org))     | [FAIRsharing](https://fairsharing.org/FAIRsharing.rs2815) 	| [re3data](https://www.re3data.org/repository/r3d100010910) 	|
|     | Protein Data Bank of Europe ([PDBe](https://www.ebi.ac.uk/pdbe/))     | [FAIRsharing](https://fairsharing.org/FAIRsharing.26ek1v) 	| [re3data](https://www.re3data.org/repository/r3d100010538) 	|
|  	| Biological Magnetic Resonance Data Bank ([BMRB](https://bmrb.io/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.p06nme) 	| [re3data](https://www.re3data.org/repository/r3d100010191) 	|
| **Electron microscopy data** 	| Electron Microscopy Data Bank ([EMDB](https://www.ebi.ac.uk/emdb/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.651n9j) 	| [re3data](https://www.re3data.org/repository/r3d100010562) 	|
|  	| Electron Microscopy Public Image Archive ([EMPIAR](https://www.ebi.ac.uk/empiar/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.dff3ef) 	| [re3data](https://www.re3data.org/repository/r3d100012356) 	|
| **Microbiome data** 	| [MGnify](https://www.ebi.ac.uk/metagenomics/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.dxj07r) 	| [re3data](https://www.re3data.org/repository/r3d100011192) 	|
| **Nucleic acid sequences** 	| [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.9kahy4) 	| [re3data](https://www.re3data.org/repository/r3d100010528) 	|
|  	| DNA Data Bank of Japan ([DDBJ](https://www.ddbj.nig.ac.jp/index-e.html)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.a308a0) 	| [re3data](https://www.re3data.org/repository/r3d100010218) 	|
|  	| European Nucleotide Archive ([ENA](https://www.ebi.ac.uk/ena/browser/home)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.dj8nt8) 	| [re3data](https://www.re3data.org/repository/r3d100010527) 	|
|  	| Sequence Read Archive ([SRA](https://www.ncbi.nlm.nih.gov/sra)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.g7t2hv) 	| [re3data](https://www.re3data.org/repository/r3d100010775) 	|
|  	| Genome Sequence Archive ([GSA](https://ngdc.cncb.ac.cn/gsa/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.tdhkc6) 	| [re3data](https://www.re3data.org/repository/r3d100012342) 	|
| **Genetic polymorphism** 	| European Variation Archive ([EVA](https://www.ebi.ac.uk/eva/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.6824pv) 	| [re3data](https://www.re3data.org/repository/r3d100011553) 	|
| **Functional genomics** 	| Gene Expression Omnibus ([GEO](https://www.ncbi.nlm.nih.gov/geo/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.5hc8vt) 	| [re3data](https://www.re3data.org/repository/r3d100010283) 	|
|  	| [ArrayExpress](https://www.ebi.ac.uk/biostudies/arrayexpress) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.6k0kwd) 	| [re3data](https://www.re3data.org/repository/r3d100010222) 	|
|  	| [GenomeRNAi](http://www.genomernai.org/) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100011089) 	|
|  	| European Genome-phenome Archive ([EGA](https://ega-archive.org/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.mya1ff) 	| [re3data](https://www.re3data.org/repository/r3d100011242) 	|
|  	| Database of Interacting Proteins ([DIP](https://dip.doe-mbi.ucla.edu/dip/Main.cgi)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.qje0v8) 	| [re3data](https://www.re3data.org/repository/r3d100010670) 	|
|  	| [IntAct](https://www.ebi.ac.uk/intact/home) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.d05nwx) 	| [re3data](https://www.re3data.org/repository/r3d100010671) 	|
|  	| Japanese Genotype-phenotype Archive ([JGA](https://www.ddbj.nig.ac.jp/jga/index-e.html)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.pwgf4p) 	| [re3data](https://www.re3data.org/repository/r3d100012689) 	|
|  	| [PubChem](https://pubchem.ncbi.nlm.nih.gov/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.qt3w7z) 	| [re3data](https://www.re3data.org/repository/r3d100010129) 	|
|  	| Genomic Expression Archive ([GEA](https://www.ddbj.nig.ac.jp/gea/index-e.html)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.hESBcy) 	| [re3data](https://www.re3data.org/repository/r3d100013187) 	|
|  	| Genome-Wide Association Studies ([GWAS](https://www.ebi.ac.uk/gwas/)) Catalog 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.blUMRx) 	| / 	|
| **Protein sequences** 	| [UniProt](https://www.uniprot.org/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.s1ne3g) 	| [re3data](https://www.re3data.org/repository/r3d100010357) 	|
| **Proteomes** 	| PRoteomics IDEntifications ([PRIDE](https://www.ebi.ac.uk/pride)) Archive database 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.e1byny) 	| [re3data](https://www.re3data.org/repository/r3d100010137) 	|
| **Quantitative and predictive food microbiology** 	| [ComBase](https://combase.errc.ars.usda.gov/) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100010878) 	|
| **Scientific texts and data** 	| [PUBLISSO – Repository for Life Sciences](https://repository.publisso.de/) 	| [FAIRsharing](https://fairsharing.org/4357) 	| [re3data](https://www.re3data.org/repository/r3d100013523) 	|
| **Species interaction data** 	| Global Biotic Interactions ([GloBI](https://www.globalbioticinteractions.org/)) 	| / 	| / 	|
| **Standardized bacterial information** 	| Bacterial Diversity Metadatabase ([BacDive](https://bacdive.dsmz.de/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.aSszvY) 	| [re3data](https://www.re3data.org/repository/r3d100013060) 	|
| **Vertebrate-virus network** 	| [VIRION](https://viralemergence.github.io/virion/) 	| / 	| / 	|
{: .table .table-hover}

## Data publishing in the PUBLISSO – Repository for Life Sciences (FRL)

#### Repository details
* **Data types**: scientific texts and research data from the fields of medicine, health, nutritional, environmental and agricultural sciences
* **Open Access**: yes
* **Terms of use & License**: yes
* **PID system**: yes (DOI)
* **Certification or repository standard**: no
* **Policy**: yes ( see "[Data Policy](https://www.publisso.de/fileadmin/user_upload/Downloads/Data_Policy_FD_Fachrepositorium_Lebenswissenschaften_Okt_2019.pdf)", only in German)
* **Archiving**: yes
* **Costs**: free of charge
* **Data curation & quality control**: no
* **Data guarantee**: availability for at least 10 years and transfer to a long-term archive
* **Recommended publication formats for research data**: see "Recommended preservation formats for research data" on this [page](https://www.publisso.de/en/research-data-management/rd-archiving/)

#### Steps to deposit data in the FRL
1.	Contact [forschungsdaten@zbmed.de](forschungsdaten@zbmed.de).
2.	You will receive:
    * The terms of use that you need to sign and send back.
    * A record sheet that you need to fill in with your metadata and send back.
3.	Your metadata will be checked, entered into the FRL and assigned a DOI.
4.	Send your data and supplementary materials to the FRL team.
5.	Your data will be added to the metadata record already created in the FRL.
6.	You will receive a publication notice with the DOI.

#### Benefits
* The FRL provides **Open Access** to its data (in specific cases, you can set an **embargo** period of up to 24 months).
* There are **no charges** to publish, archive or use scientific texts and research data from the FRL.
* The FRL supports the [**FAIR data principles**](https://www.go-fair.org/fair-principles/) (e.g. by assigning DOIs, by offering life science-specific metadata).
* You can publish your **metadata** in the FRL and receive a **DOI** in advance of publishing your data.
* You can publish **large datasets** in the FRL.
* The FRL is **permanently accessible** online.
* The FRL is indexed in [**re3data**](https://www.re3data.org/repository/r3d100013523).
* The FRL is **findable** via Google search and indexed in [BASE](https://www.base-search.net/about/en/index.php), [DataCite Search](https://search.datacite.org/) and  [LIVIVO - ZB MED Search Portal for Life Sciences](https://www.livivo.de/).

#### Drawbacks
* The FRL is not suitable for sensitive data.
* The FRL can be browsed only in German.

## Data publishing in Zenodo

#### Repository details
* **Data types**: all research outputs, positive and negative results
* **Open Access**: yes
* **Terms of use & License**: yes
* **PID system**: yes (DOI)
* **Certification or repository standard**: no
* **Policy**: yes (see [here](https://about.zenodo.org/policies/))
* **Archiving**: yes
* **Costs**: free of charge
* **Data curation & quality control**: by the repository itself or by communities
* **Data guarantee**: availability for the lifetime of the repository
* **Recommended publication formats for research data**: any file format

#### Steps to deposit data in Zenodo 
1.	Upload files.
2.	Describe your content so others can find it.
3.	Publish your content.

For more details, see this [guide](https://www.openaire.eu/zenodo-guide).

#### Benefits
* Zenodo provides **Open Access** to its data (files may be deposited under closed, open or embargoed access).
* There are **no charges** to publish, archive or use content.
* Zenodo supports the [**FAIR data principles**](https://www.go-fair.org/fair-principles/) (e.g. by assigning DOIs).
* Zenodo is **permanently accessible** online.
* Zenodo is indexed in [**re3data**](https://www.re3data.org/repository/r3d100013523).
* Zenodo is **findable** via Google search and indexed in [BASE](https://www.base-search.net/about/en/index.php) and [DataCite Search](https://search.datacite.org/).

#### Drawbacks
* Zenodo might not be suitable for sensitive data that might have to stay in their country of origin.
* Zenodo has a size limit of 50 GB per record. For bigger size, a request has to be made.

## Repository finders
* To find any repositories:
    * The DFG-funded registry of research data repositories ([re3data.org](https://www.re3data.org/), [search results](https://www.re3data.org/search?query=Microbiology) for "Microbiology")
    * The RDA-endorsed [FAIRsharing.org](https://fairsharing.org/) ([search results](https://fairsharing.org/search?fairsharingRegistry=Database&subjects=microbiology&page=1) for Databases in Microbiology)
* To find Open Access repositories in the life sciences: [ELIXIR Deposition Databases for Biomolecular Data](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
* To find a suitable interdisciplinary repository: [Generalist Repository Comparison Chart](https://doi.org/10.5281/zenodo.3946720)
* To find Open Access repositories: [OpenDOAR](https://v2.sherpa.ac.uk/opendoar/): Directory of Open Access Repositories

## See Also
* [Data Deposition and Standardization](https://academic.oup.com/nar/pages/data_deposition_and_standardization) help page of the [Oxford Academic](https://academic.oup.com) Nucleic Acids Research ([NAR Journal](https://academic.oup.com/nar)).

# Get Help
If you have any further questions about the management and analysis of your microbial research data, please contact us: [helpdesk@nfdi4microbiota.de](mailto:helpdesk@nfdi4microbiota.de) (by emailing us you agree to the [privacy policy - in German](https://nfdi4microbiota.de/legals/privacy-policy.html) on our website: [Contact](https://nfdi4microbiota.de/contact-form/). The [legaly non-binding English translation can be found here]({% link _Privacy-Policy-English-Translation/01-Privacy-Policy-English-Translation.md %}) )

## References
* Engelhardt, C., Biernacka, K., Coffey, A., Cornet, R., Danciu, A., Demchenko, Y., Downes, S., Erdmann, C., Garbuglia, F., Germer, K., Helbig, K., Hellström, M., Hettne, K., Hibbert, D., Jetten, M., Karimova, Y., Kryger Hansen, K., Kuusniemi, M. E., Letizia, V., … Zhou, B. (2022). D7.4 How to be FAIR with your data. A teaching and training handbook for higher education institutions (V1.2.1). Zenodo. [https://doi.org/10.5281/ZENODO.6674301](https://doi.org/10.5281/ZENODO.6674301)
* Lindlar, M., Rudnik, P., Horton, L., & Jones, S. (2020). “You say potato, I say potato” - Mapping Digital Preservation and Research Data Management Concepts towards Collective Curation and Preservation Strategies. [https://doi.org/10.5281/ZENODO.3672773](https://doi.org/10.5281/ZENODO.3672773)
* Rathmann T. et al. 2021-10. Workshop on Research Data Management. FoDaKo and ZB MED - Information Centre for Life Sciences. Google Slides.
